src/main/java/bio/terra/app/controller/CohortBuilderApiController.java

```diff
@@ -0,0 +1,432 @@
+                                                                .dataType(DataType.INT64)
+                                                                .valueUnion(
+                                                                    new LiteralValueUnion()
+                                                                        .int64Val(38003563L))))
+                                                .count(54453),
+                                            new DisplayHintEnumEnumHintValues()
+                                                .enumVal(
+                                                    new ValueDisplay()
+                                                        .display("Not Hispanic or Latino")
+                                                        .value(
+                                                            new Literal()
+                                                                .dataType(DataType.INT64)
+                                                                .valueUnion(
+                                                                    new LiteralValueUnion()
+                                                                        .int64Val(38003564L))))
+                                                .count(2272403))))))
+            .addDisplayHintsItem(
+                new DisplayHint()
+                    .attribute(
+                        new Attribute()
+                            .name("year_of_birth")
+                            .type(Attribute.TypeEnum.SIMPLE)
+                            .dataType(DataType.INT64))
+                    .displayHint(
+                        new DisplayHintDisplayHint()
+                            .numericRangeHint(
+                                new DisplayHintNumericRange().min(1909.0D).max(1983.0D))));
+
+    return new ResponseEntity<>(new DisplayHintList(), HttpStatus.NOT_IMPLEMENTED);
+  }
+
+  // Our stubs have the same count for everything
+  private List<InstanceCount> createInstanceCountsYearGenderRace(
+      List<ValueDisplay> genders, List<ValueDisplay> races, List<ValueDisplay> yearsOfBirth) {
+    List<InstanceCount> instanceCounts = new ArrayList<>();
+    genders.forEach(
+        (ValueDisplay gender) ->
+            races.forEach(
+                race ->
+                    yearsOfBirth.forEach(
+                        (ValueDisplay yearOfBirth) -> {
+                          instanceCounts.add(
+                              new InstanceCount()
+                                  .count(20000)
+                                  .attributes(
+                                      Map.of(
+                                          "gender",
+                                              new ValueDisplay()
+                                                  .value(gender.getValue())
+                                                  .display(gender.getDisplay()),
```

Comment:
Would it not work to pass `gender` here?
```suggestion
                                              gender,
```

Reply:
No reason it wouldn't, good catch!